packages/aws-cdk/lib/cli.ts

```diff
@@ -294,6 +295,9 @@ async function initCommandLine() {
   if (argv.ci) {
     setCI(true);
   }
+
+  await checkForPlatformWarnings();
```

Comment:
Should we catch any errors and just proceed?

Reply:
Shouldn't happen, but sure. Sounds safe.